src/App.tsx

```diff
@@ -1,8 +1,58 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
 import './App.css';
+import { PostsList } from './components/PostsList/PostsList';
+import { getPosts, getUsers, getComments } from './utils/api';
 
-const App = () => (
-  <h1>Dynamic list of posts</h1>
-);
+import {
+  FullPostType,
+  PostType,
+  UserType,
+  CommentType,
+} from './utils/interfaces';
+
+const App: FC = () => {
+  const [posts, setPosts] = useState<FullPostType[]>([]);
+  const [isLoading, setLoading] = useState(false);
+
+  const showPosts = async () => {
+    setLoading(true);
+
+    const [loadedPosts, loadedUsers, loadedComments] = await Promise
+      .all([getPosts(), getUsers(), getComments()]);
+
+    const fullPosts = loadedPosts.map((post: PostType) => ({
```

Comment:
если описать что отдает getPosts, то тут это не нужно
```suggestion
    const fullPosts = loadedPosts.map((post) => ({
```